packages/web-components/src/design-tokens.ts

```diff
@@ -2,10 +2,14 @@ import { DesignToken } from '@microsoft/fast-foundation';
 
 const { create } = DesignToken;
 
+// ----------------------------------------
+// Border Radius
+// ----------------------------------------
+
 /**
  * @public
  */
-export const borderRadiusNone = create<number>('borderRadiusNone').withDefault(0);
+export const borderRadiusNone = create<string>('borderRadiusNone').withDefault('0');
```

Comment:
Thanks for this - I was wondering how to best handle this as it's an odd case to override a token for "none" with anything but zero, but it makes sense.